src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -52,7 +52,7 @@ public void onCommand(Member sender, CommandContext context) {
             statusName = FormatUtils.formatEnum(member.getOnlineStatus());
             if (member.getGame() != null && member.getGame().getType() == Game.GameType.STREAMING) {
                 status = context.globalEmote("streaming");
-                statusName = "Streaming";
+                statusName = context.i18n("commands.userinfo.streaming");
```

Comment:
This will probably be very commonly used across the bot, put all the different statuses in a separate section